### PR TITLE
Local signatures

### PR DESCRIPTION
**Pull request by almet**
This allows to do signatures from one collection to the other one.

Status:

- [x] Implement the feature
- [x] Update the function tests
- [ ] Handle permissions on the destination collection / bucket
- [x] Update the unit tests
- [x] Update the README

### CODE REVIEW

**Comment by almet** [comment]
Not sure, but we might also want to use the `cliquet.resource.model` classes rather than querying the storage backend ourselves, to investigate, and maybe in a later pull request.

File: kinto.ini [hunk 1]
```diff
@@ -5,15 +5,18 @@ port = 8888
...
 
 kinto.includes = kinto_signer.hook
 
 ; Kinto Updater configuration.
 ; XXX Put this in a section instead.
-kinto_signer.bucket =
-kinto_signer.collection =
+
+kinto_signer.resources =
+    local:bucket/collection, remote:bucket/collection
```
Comment (Natim) [inline kinto.ini]: This should be "bucket/collection;signed/collection"

File: kinto_signer/hook.py [hunk 2]
```diff
@@ -1,60 +1,73 @@
...
             return
 
+        resource = available_resources.get(requested_resource)
         should_sign = any([True for r in event.impacted_records
                            if r['new'].get('status') == 'to-sign'])
         if should_sign:
             registry = event.request.registry
-            updater = RemoteUpdater(
-                remote=remote,
+            # XXX Add Auth.
```
Comment (Natim) [inline kinto_signer/hook.py]: We can remove this XXX

File: kinto_signer/tests/config/signer.ini [hunk 3]
```diff
@@ -13,10 +13,11 @@ kinto.flush_endpoint_enabled = true
...
 
-kinto_signer.bucket = buck
-kinto_signer.collection = coll
-kinto_signer.remote_server_url = http://localhost:7777/v1
-kinto_signer.remote_server_auth = user:p4ssw0rd
+kinto_signer.resources =
+    source/collection1;destination/collection1
+    source/collection2;destination/collection2
+
+kinto_signer.remote = http://user:p4ssw0rd@localhost:7777/v1
```
Comment (Natim) [inline kinto_signer/tests/config/signer.ini]: We can remove this option.

**Comment by Natim** [comment]
A soon as travis is green, I think we should merge this.

I have created the related missing parts as separate issues #25 #26 #27

**Pull request merged**